proto/sedachain/wasm_storage/v1/events.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+package sedachain.wasm_storage;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+import "sedachain/wasm_storage/v1/wasm_storage.proto";
+
+option go_package = "github.com/sedaprotocol/seda-chain/x/wasm-storage/types";
+
+
+message EventDataRequestWasm {
+  string hash = 1;
+  WasmType wasm_type = 2;
+
+}
+
```

Comment:
An alternative could be: 
`message EventWasm {
  string hash = 1;
  WasmType wasm_type = 2;
   RequestType request_type = 3;

}`

@hacheigriega what do you think?

Reply:
The module would be shown in the proto-type. This is an example of group modue event
<img width="590" alt="image" src="https://github.com/sedaprotocol/seda-chain/assets/8561557/35a81dc4-98a6-4cec-81d7-a90807873ca4">